presto-docs/src/main/sphinx/release/release-344.md

```diff
@@ -0,0 +1,64 @@
+# Release 344 (XX Oct 2020)
+
+## General Changes
+
+* Add {func}`murmur3` function. ({issue}`5054`)
+* Add {func}`from_unixtime_nanos` function. ({issue}`5046`)
+* Add [T-Digest](/functions/tdigest) type and functions. ({issue}`5158`)
+* Improve performance and latency of queries leveraging dynamic filters. ({issue}`5081`)
+* Allow to configure (using ``dynamic-filtering.service-thread-count`` feature toggle) number of
+  threads used for processing dynamic filters on coordinator. ({issue}`5341`)
+* Extend environment variables substitution (`${ENV:variable-name}`) to work also when the substituted text
+  is part of the configuration value. ({issue}`4345`)  
+* Remove `fast-inequality-joins` configuration property. The fast inequality joins remain enabled. ({issue}`5375`)
+* Report dynamic filters collection duration. ({issue}`5434`)  
```

Comment:
report.. where?

Reply:
`Report dynamic filters collection duration as part of query statistics JSON`?

Reply:
That's internal. The JSON is not meant to be consumed by users. So unless this is being surfaced in the UI or CLI, I'd just remove it.

Reply:
We can remove it. It's described in documentation too BTW